node/service/src/chain_spec.rs

```diff
@@ -455,16 +580,179 @@ fn kusama_staging_testnet_config_genesis(wasm_binary: &[u8]) -> kusama::GenesisC
 		pallet_babe: Some(Default::default()),
 		pallet_grandpa: Some(Default::default()),
 		pallet_im_online: Some(Default::default()),
-		pallet_authority_discovery: Some(kusama::AuthorityDiscoveryConfig {
-			keys: vec![],
-		}),
+		pallet_authority_discovery: Some(kusama::AuthorityDiscoveryConfig { keys: vec![] }),
 		claims: Some(kusama::ClaimsConfig {
 			claims: vec![],
 			vesting: vec![],
 		}),
-		pallet_vesting: Some(kusama::VestingConfig {
-			vesting: vec![],
+		pallet_vesting: Some(kusama::VestingConfig { vesting: vec![] }),
+	}
+}
+
+fn rococo_staging_testnet_config_genesis(wasm_binary: &[u8]) -> rococo::GenesisConfig {
```

Comment:
I don't understand why you not copied the old function...

Reply:
I'm fairly sure that I copied this from _somewhere_. If there's a better source, I'm happy to update it.

Reply:
You did not copy it from Rococo. This is westend or whatever.

Reply:
Hmm. Not sure where I should have copied it from. The function doesn't exist in [`rococo-branch`](https://github.com/paritytech/polkadot/blob/rococo-branch/node/service/src/chain_spec.rs) or [`bkchr-rococo`](https://github.com/paritytech/polkadot/blob/bkchr-rococo/node/service/src/chain_spec.rs) [`cumulus-branch`](https://github.com/paritytech/polkadot/blob/cumulus-branch/node/service/src/chain_spec.rs).

Reply:
You should check the correct file: https://github.com/paritytech/polkadot/blob/26f1fa47f7836ab4bee5d4aad127ebce748320dd/service/src/chain_spec.rs#L362

Reply:
Updated in 34e51e5e.